src/main/kotlin/com/codibly_doko/alarm/model/Alarm.kt

```diff
@@ -0,0 +1,18 @@
+package com.codibly_doko.alarm.model
+
+import com.codibly_doko.common.model.ChartType
+import com.codibly_doko.user.model.User
+import java.time.Instant
+import java.util.*
+import javax.persistence.*
+
+@Entity
+data class Alarm(
+    @Id @GeneratedValue var id: UUID?,
+    @ManyToOne @JoinColumn(name = "user_id", nullable = false) val user: User,
+    var name: String,
+    var description: String?,
+    var value: Double,
+    var lastOccurred: Instant?,
```

Comment:
can we use `val` here?

Reply:
Only in immutable, that we cannot update, (user, chartType)

Reply:
It's a data class, you can use `val` everywhere and use it's `copy` function, like this:
```
alarm.copy(name = "Name")
```